### PR TITLE
[Behat 2x] Checks on redirects response

### PR DESCRIPTION
**Pull request by gmorel**
#### Feature 

Allow not following redirects in order to check directly redirect response 

By adding one time toggle 
```gherkin
Given I won't follow next redirect
```

#### Example

```gherkin
Given I won't follow next redirect
When I send a GET request to "/redirecting-uri.json"
Then response status code should be 301
And response header "Location" should have value "/redirected_uri.json"
```

### CODE REVIEW

File: src/RestApiContext.php [hunk 1]
```diff
@@ -12,6 +12,7 @@
 
 class RestApiContext extends BehatContext implements JsonStorageAware
 {
+    /** @var \mageekguy\atoum */
```
Comment (gmorel) [inline src/RestApiContext.php]: Allow IDE autocomplete

**Comment by gmorel** [comment]
If good for you. I make the same for Behat 3 branch.

**Comment by shouze** [comment]
LGTM @gmorel you have UT now... nothing to add?

**Comment by gmorel** [comment]
Need to test on our project behat suite to validate no regression then OK

**Pull request merged**